src/swagger.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
Perhaps the swagger annotations can be derived from registered routes/classes on $app

Then read some sort of annotation from each route class?